keras_cv/layers/__init__.py

```diff
@@ -135,6 +135,9 @@
 )
 from keras_cv.layers.spatial_pyramid import SpatialPyramidPooling
 from keras_cv.layers.transformer_encoder import TransformerEncoder
+from keras_cv.layers.video_swin_layers import VideoSwinBasicLayer
+from keras_cv.layers.video_swin_layers import VideoSwinPatchingAndEmbedding
+from keras_cv.layers.video_swin_layers import VideoSwinPatchMerging
```

Comment:
We probably don't want these in the public API, just internal layers for `VideoSwinTransformer`. What do you think? Was there a reason to make them public?

Reply:
There is no reason to place them here. I was just keeping the consistency with other component.  For example, all of the `vit_det_layers` are placed here. Anyway, will remove it.

Reply:
It was a mistake to export the `vit_det_layers`. Other than something like `WindowPartitioning`, other layers are too specific for the detectron2 ViT model.

It would be nice to keep the layers API clean. Thanks for removing these. We can always add again if there's a usecase for it elsewhere!

Reply:
Got it. And note, I'm removing imports that I added in this PR. Others import, like `vit_det` or anything else, let's do that later (coz some evaluation is needed of their presence here).